float32 num1
float32 num2
float32 id
---
float32 result

